### PR TITLE
Migrated plugin to Nativescript 7

### PR DESCRIPTION
**Pull request by nvukovic**

### CODE REVIEW

File: demo/app/main-page.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { EventData } from 'tns-core-modules/data/observable';
-import { Label } from 'tns-core-modules/ui/label';
-import { Page } from 'tns-core-modules/ui/page';
-import * as frameModule from "tns-core-modules/ui/frame";
+import { EventData } from '@nativescript/core';
+import { Label } from '@nativescript/core';
+import { Page } from '@nativescript/core';
```
Comment (b-ivanovic) [inline demo/app/main-page.ts:3]: Probably would be nicer if you group all @nativescript/core imports into one.

File: demo/webpack.config.js [hunk 2]
```diff
@@ -1,321 +1,339 @@
...
-    }
+	// Default destination inside platforms/<platform>/...
+	const dist = resolve(projectRoot, nsWebpack.getAppPath(platform, projectRoot));
 
-    // Default destination inside platforms/<platform>/...
-    const dist = resolve(projectRoot, nsWebpack.getAppPath(platform, projectRoot));
+	const {
+		// The 'appPath' and 'appResourcesPath' values are fetched from
+		// the nsconfig.json configuration file.
+		appPath = 'src',
```
Comment (b-ivanovic) [inline demo/webpack.config.js:35]: Should it not be 
appPath = 'app',
appResourcePath = 'app/App_Resources',

in your case?

File: demo/app/main-page.ts (hunk 1 above)
Comment (nvukovic) [reply demo/app/main-page.ts:3]: I agree, it will be updated.

File: demo/webpack.config.js (hunk 2 above)
Comment (nvukovic) [reply demo/webpack.config.js:35]: The appPath and appResourcePath should be picked up from nativescript.config.ts from NS7. Aldo, I will chage it here just to not have confusion.

**Review by b-ivanovic — APPROVED** [review]

**Pull request merged**